.gitignore

```diff
@@ -1,30 +1,83 @@
-# Compiled source #
```

Comment:
Hey, is there any way you can pull back the pages that were not changed? I have to sort thru every different page if git all is used.